source/PlayServicesResolver/src/PlayServicesResolver.cs

```diff
@@ -928,32 +926,69 @@ public class AndroidSdkRootChangedArgs : EventArgs {
         /// Defaults to 1 second.</param>
         private static void ScheduleAutoResolve(double delayInMilliseconds = 1000.0) {
             lock (typeof(PlayServicesResolver)) {
-                if (!autoResolving) {
-                    RunOnMainThread.Cancel(autoResolveJobId);
-                    autoResolveJobId = RunOnMainThread.Schedule(
-                        () => {
-                            lock (typeof(PlayServicesResolver)) {
-                                autoResolving = true;
-                            }
-                            RunOnMainThread.PollOnUpdateUntilComplete(() => {
-                                if (EditorApplication.isCompiling) return false;
-                                // Only run AutoResolve() if we have a valid autoResolveJobId.
-                                // If autoResolveJobId is 0, ScheduleResolve()
-                                // has already been run and we should not run AutoResolve()
-                                // again.
-                                if (autoResolveJobId != 0) {
-                                    AutoResolve(() => {
-                                        lock (typeof(PlayServicesResolver)) {
-                                            autoResolving = false;
-                                            autoResolveJobId = 0;
+                if (autoResolving)
+					return;
+
+				RunOnMainThread.Cancel(autoResolveJobId);
```

Comment:
style: indentation looks off here.  In general there are a lot of whitespace changes in this commit series.  Any chance you could:
- Squash the commit series into a single commit?
- Only apply whitespace changes in areas where you've modified how the code work?

That will make it much easier to see what's going on here.